scripts/release.sh

```diff
@@ -0,0 +1,47 @@
+#! /bin/bash
+
+# COLORS
+REDBOLD='\033[1;31m'
+RED='\033[0;31m'
+PURPLE='\033[0;35m'
+TEAL='\033[0;36m'
+GREEN='\033[1;32m'
+
+gitMessage=$(git log --oneline -n 1)
+mainBranch="master"
+branch=$(git rev-parse --abbrev-ref HEAD)
+
+echo -e "${PURPLE}Checking all branches are up-to-date..."
+echo -e "================================================="
+echo -e " git fetch --all"
+git fetch --all
+echo -e " git pull origin ${mainBranch}"
+git pull origin $mainBranch
```

Comment:
not sure if pull is what's desired here, since a) that does a fetch that's already done, and b) that merges, when we probably just want `git checkout origin/$mainBranch` instead of being on a branch at all?